pkgs/development/python-modules/conjure-python-client/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  requests,
+}:
+buildPythonPackage rec {
+  version = "3.0.0";
+  pname = "conjure-python-client";
+  pyproject = true;
+
+  src = fetchPypi {
```

Comment:
`fetchFromGitHub` is preferred

Reply:
I had some trouble packaging this from source so went with pypi instead. I can thread those problems here for discussion if it's _important_ to not use pypi.

Reply:
Sure, maybe I can help make it work.

Reply:
Thanks for the offer! I got it to work though.